server/controllers/user-controller.js

```diff
@@ -0,0 +1,81 @@
+import {
+  successResponse,
+  errorResponse,
+  validate,
+  validationErrorResponse,
+  checkUniqueUserName
+} from '../utils/helpers';
+import db from '../models';
+import { profileSchema } from '../utils/validators';
+
+const { User } = db;
+
+/**
+ * The controllers for users route
+ *
+ * @class UsersController
+ */
+class UsersController {
+  /**
+   * Create user profile
+   * @static
+   * @param {Request} req request object
+   * @param {Response} res response object
+   * @memberof {Users}
+   * @returns {undefined}
+   */
+  static async updateUser(req, res) {
+    validate(req.body, profileSchema)
+      .then(async () => {
+        const { id } = req.user;
+        try {
+          const { username, bio, image } = req.body;
+          const user = await User.findOne({
+            where: { id }
+          });
+          const isUnique = await checkUniqueUserName(username, id);
```

Comment:
I thought I had commented on this, no? Why is there a check in code that the username is unique - shouldn't that be handled by the database?